src/components/organisms/ActionsPane/ActionsPane.tsx

```diff
@@ -317,13 +317,14 @@ const ActionsPane = (props: {contentHeight: string}) => {
   }, [selectedResourceId, resourceMap]);
 
   useEffect(() => {
-    if (
-      (activeTabKey === 'metadataForm' || activeTabKey === 'form') &&
-      (!selectedResourceId || !(isKustomization || (resourceKindHandler && resourceKindHandler.formEditorOptions)))
-    ) {
+    if (activeTabKey === 'form' && !isKustomization && !resourceKindHandler?.formEditorOptions?.editorSchema) {
```

Comment:
hm.. shouldn't this be 

```
if (activeTabKey === 'form' && (!isKustomization || !resourceKindHandler?.formEditorOptions?.editorSchema)) {
```

!?

Reply:
this code is redundant

```
isKustomization ? (
                    <FormEditor formSchema={extractFormSchema(getResourceSchema(selectedResource))} />
                  ) : (
                    resourceKindHandler?.formEditorOptions && (
                      <FormEditor
                        formSchema={resourceKindHandler.formEditorOptions.editorSchema}
                        formUiSchema={resourceKindHandler.formEditorOptions.editorUiSchema}
                      />
                    )
``` 
because ` getResourceSchema`  query the resource kind handler and returns the editor schema

Reply:
the issue is that kustomizations don't have kindhandlers - but they still have forms/schemas (we will fix this in a future release) - which is why we must check both...